### ISSUE TITLE
Two clients for two different teams: InvalidProviderToken

### ISSUE THREAD

**Issue by edvinasbartkus** [AndrewBarba/apns2#90]
We have done app migration so we have a scenario where TestFlight is using Apple Developer Account 1 and production app is using Apple Developer Account 2. In code, we have the following:

```js
const client = new ApnsClient({
    team: 'team1',
    keyId: 'key1',
    signingKey: process.env.APNS_KEY_CLIENT1,
    defaultTopic: 'bundle1',
    requestTimeout: 0,
    keepAlive: true
})

const client2 = new ApnsClient({
    team: 'team2',
    keyId: 'key2',
    signingKey: process.env.APNS_KEY_CLIENT2,
    defaultTopic: 'bundle2',
    requestTimeout: 0,
    keepAlive: true
})

function process() {
  ...
  if (notification.team == 'team1') {
    client.send(payload)
  } else {
    client2.send(payload)
  }
}
```

Since we process a large batch of notifications at once, we encounter that attempt to use client2 results in error 403 Forbidden: {"reason":"InvalidProviderToken"}

In the following [StackOverflow question](https://stackoverflow.com/a/45753393) I have learned that there is issue with APNs: in order to switch to different team a new connection must be established.

Wanted to double check here is it so that apns2 library is reusing connections to apns server regardless of number of clients created in code?

**Comment by AndrewBarba** [comment]
Yes I think that's the case. I'm working on a fix

**Comment by edvinasbartkus** [comment]
I did solve by having dedicated lambda functions to deliver the messages for different teams. No issue on my side at the moment.

Thank you for your great work on apns2!

**Comment by AndrewBarba** [comment]
Here's the fix @edvinasbartkus https://github.com/AndrewBarba/apns2/commit/4b91ef00f389423912d43f3bcb5021022284b782

**Comment by AndrewBarba** [comment]
Released as 11.8.0 https://github.com/AndrewBarba/apns2/releases/tag/11.8.0
